index.Rmd

```diff
@@ -29,7 +29,7 @@ To request an R mentor please complete [this mentee form](https://forms.office.c
 This internal training is recommended for those working in Data and Analysis as it enables you to get up and running with R, SQL, Python and GitHub using the MoJ Analytical Platform. The main introductory R, SQL and GitHub sessions are run live in February/March, June/July and October/November each year while you can also work through R, SQL and Python sessions yourself using the training material and/or recordings. To learn more about the sessions currently available and how to access the material and recordings, go to the [Internal Training Group materials](#ITG) chapter.
 
 ## Coffee and Coding and Bitesize sessions
```

Comment:
Maybe split this into two headings, one for C&C and one for the bitesize sessions?

Reply:
Good idea, done!